_treehouses

```diff
@@ -362,6 +362,11 @@ treehouses wifihidden
 treehouses wifistatus
 treehouses convert
 treehouses resolution
+treehouses editor default
+treehouses editor set vim
+treehouses editor set emacs
+treehouses editor set nano
+treehouses editor config
```

Comment:
You should add `treehouses editor`.
Also it would be great if you could alphabetize the list.